tests/test_api.py

```diff
@@ -40,22 +40,25 @@ def test_happy_path(self, access_token, url):
         bundle_url = bundleObj["meta"]["profile"][0]
         response_number_of_entries = len(bundleObj["entry"])
 
-        resourceType_operationoutcome = bundleObj["entry"][28]["resource"][
+        resourceType_operationoutcome = bundleObj["entry"][30]["resource"][
```

Comment:
Using these indices without comments regarding what exactly they are is confusing, we use the `json_rw` library to parse data as structured python objects in the main code, would that be useful here?